shared/chat-sdk/reference/release-notes/windows.mdx

```diff
@@ -1,5 +1,68 @@
 <PlatformWrapper platform="windows">
 
+## v1.0.9
+
+v1.0.9 was released on February 1, 2023.
+
+#### New features
+
+1. Adds the following methods to the `SDKClient` class:     
+  - `GetLoggedInDevicesFromServer`: Gets the list of online devices on which you have logged in with a specified account.
+  - `KickDevice`: Logs out from a specified account on a device.
+  - `kickAllDevices`: Logs out from a specified account on all devices.
+2. Adds the following methods to the `RoomManager` class: 
+  - `FetchAllowListFromServer`: Gets the allow list of a chat room from the server.
+  - `CheckIfInRoomAllowList`: Checks whether the current member is on the chat room block list.
```

Comment:
Just checking here whether block list and mute list are the same of different things? If same, please pick one name